### DISCUSSION TITLE
Welcome to Alpas Discussions!

### DISCUSSION THREAD

**Discussion by ashokgelal** [alpas/alpas#43] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by egorsmkv** [comment]
Hello!

**Comment by fklappan** [comment]
Quick question: what is the current state of this project? Unfortunately it seems pretty dead, but I really like the idea of an "batteries included" Kotlin web framework.

Do you have any future plans? Last commits are pretty old apart from the M1 branch (which are obviously very specific changes). The last release was version 0.16 from march 2020.

Happy coding!
